js/runner.go

```diff
@@ -159,6 +197,17 @@ func (r *Runner) newVU(samplesOut chan<- stats.SampleContainer) (*VU, error) {
 		Blacklist: r.Bundle.Options.BlacklistIPs,
 		Hosts:     r.Bundle.Options.Hosts,
 	}
+	if r.Bundle.Options.Nic.Valid {
+		randAddr, err := getRandomIP(r.Bundle.Options.Nic.ValueOrZero())
+		if err != nil {
+			return nil, err
+		}
+
+		localTcpAddr := net.TCPAddr {
+			IP: randAddr.IP,
+		}
+		dialer.Dialer.localAddr = &localTcpAddr
```

Comment:
I'm getting a `dialer.Dialer.localAddr undefined` error locally, so this should be:

```suggestion
		dialer.Dialer.LocalAddr = &localTcpAddr
```

Also, to appease the linter, consider naming the var `localTCPAddr`.

Reply:
Fixed in 90bbf39bf6443feea1792eb169629e9eb61c129f.